src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -107,4 +111,42 @@ module sonic-types {
             pattern "percentage|used|free|PERCENTAGE|USED|FREE";
         }
     }
+
+    typedef oper-status {
```

Comment:
is unknown a possible value?

Reply:
Added "unknown" also as one of enum.

Reply:
@lguohan : Need suggestion, do we want to add non-config DB related Yang model in same file or we want to keep them separate?